fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -3607,6 +3607,12 @@ private static void addOlapTablePropertyInfo(OlapTable olapTable, StringBuilder
             sb.append(olapTable.rowStorePageSize()).append("\"");
         }
 
+        // storage page size
+        if (olapTable.storagePageSize() != PropertyAnalyzer.STORAGE_PAGE_SIZE_DEFAULT_VALUE) {
```

Comment:
is page size changable after creating the table.

Reply:
It cannot be modified.